FlugelTest-444a4376f135.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "flugeltest",
+  "private_key_id": "444a4376f135329ed0abf2677915ba57e5d05f76",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCz4HKd30QZrf7z\nUVkw17v9rZ5je6YUutja9kny4a0YTYbRhsK1O/sMJOYS/zsgY3KzxaA7jFDfOJBn\np6fTBK6DfNwMb+l+BJAdgKHuiZagLUCtTb51Nm8I/MS8IJ7OXQ/PgM0IRGfsV9Vr\nfhFrwBLLqH9oLnm7xvfyOroOLz4Rx9WOpzhSMQtbFyUNGkCPJUGPKX7g81jrBxl3\nh5ey24qGgOXmK6f4d6hdvD+B0H+uhvHwQbBcC7gtJxp8wje5BgTuuLISwQKrDlFY\n+Eh0xb93uMYFCrRc/ZNyKB+S+37ymQk4IiBheoJipfXtXmhq+7wngwKMDoK7R0OL\nV3Dy5m9/AgMBAAECggEAFzUcpJjDOQr1a2AyATa8Vvj5B/3SijcnNmzzUoZiZlHO\nVKct5OMlu0Xhd079ymp6SI51Y2lxn7M1EmKTqQqgF2/bFjG0LuGY1G2dtuCCkgz0\nnMd8jLPmtIKkQV92Dcr8+Lk1+tyUie41J2a/Cadg5WzKb18brqn3oIWKLr1SpUlr\nChVjPrPJWE90rqsybeHQWQ+yzOHepqXS2n00WW9kw+abWkFjq6lQ5oD5AFuiysEv\n8XsPNJ5WWQOfCs3c7PdpiZBqx0EwTcIB2YTWBg6Nwdw2/gFOqQ7+7S7nueobK7Pc\nsGz/91Luh09HD2vLg4P3gotvo2MmULDSERuzCdhmgQKBgQDo1CH5Ju7UMOxZKziE\nv24pEBuw9Qv6kTDUH389Lqb+55cODWf5uwUlabyBHwCxqRLS2B7OK4f0UuZFyTth\nNvcm7lgdtR+U8THkn3fVMFOpzfxamqgaaefufc2pCFnaRtQxsURm0UgpcO8UQND3\nOQ5GTgeSfFl+FLg5x2YA2TA9/wKBgQDFxz0al7jWBU/eqjkETiQc9z+0I4+z9MqG\nEJgj0X4vogVH40csFNEu5Zdlk3YbbPQIPu9g0Gq6Si9j3UPn0ObuzjIn5W9tAEJB\nWm3UQqBxauIxjRpTwiNvgv1KeuSUzAMT60VwaAIXetdH7PwK/Q0FwHIIDZBc3XH+\n2l6juEzOgQKBgQCKnro3aAmrxhEBqez1hC/nPNLv0YW3QjhrIvGTkLIWc/YIhMzR\n75pJwyFbUwULwZkWdVwp7dr9CVpWVFAJOjt+L0vBLDRIbi4BazxibmpxX9WOcCQD\nzegrozUOHY25mo/9oggSmym2mqmNSmXfu1WXYB+8BFMX8OgaZnCBss25KwKBgQCr\nkxqJNVIeKOGPEzD/cSKXFk9nYOpLfXMsNr0lZZoIjI/0O1EFBtIp0vcyIWj2heB2\nPtK/tE7CaZUPF2bPdkTAqpSMU9gEQZM4ytA4scwqWET+I6DRRLwlAwflUxN1mj6Z\nSyMkTrZx1BlqxkYo6jof4UGLhujUBy/bnWk4squ/AQKBgCYOiJg5OqREf3/XBbGi\n8fQHp0s/urqxZMrwwNaRTf0HRk5SiCyeMVdWQCdhUvIbH4EPHWLKQAzSiXLWr2f4\nzL6cPEAv0y/tdj4awSzWfn7jyx0TFZNk3gRFGuaFtXsi3QT6jNTIMTWxvk0aK3PD\nxLf2nD/ahbly4KfOXgXNQuHQ\n-----END PRIVATE KEY-----\n",
```

Comment:
Why do we have this private key here? It's a security issue.

Reply:
I used these credentials to access the project via provider "google". As below:

provider "google" {
  credentials = "${file("FlugelTest-444a4376f135.json")}"
  project     = "flugeltest"
  zone = "us-central1-a"
}

Reply:
I understand, but they must not be in clear text in the repo. Actually, this repo is public, so you exposed your GCP account to the world.